go/vt/mysqlctl/backupengine.go

```diff
@@ -119,6 +125,10 @@ var BackupRestoreEngineMap = make(map[string]BackupRestoreEngine)
 // This must only be called after flags have been parsed.
 func GetBackupEngine() (BackupEngine, error) {
 	name := *backupEngineImplementation
+	return getBackupEngine(name)
```

Comment:
very small method extraction to let me reuse the error codes, but specifying a particular engine name rather than "whatever value the flag pointer has"